src/publishers/mod.rs

```diff
@@ -4,8 +4,20 @@ use aws_sdk_sqs::Error;
 
 use crate::config::config;
 
-async fn send_sqs_message(queue_url: &str, message: &str) -> Result<(), Error> {
+async fn send_sqs_message_with_delay(queue_url: &str, message: &str, delay_seconds: i32) -> Result<(), Error> {
```

Comment:
can delay be negative ?

Reply:
I don't think so but the aws sdk expects an i32

Reply:
we can pass it as u32 and just cast it in the call then

Reply:
sure, fixed!